code/datums/components/item_equipped_movement_rustle.dm

```diff
@@ -0,0 +1,77 @@
+/datum/component/item_equipped_movement_rustle
+	var/default_rustle_sounds = SFX_SUIT_STEP
+
+	///sound that plays, use an SFX define if there is multiple.
+	var/rustle_sounds
```

Comment:
actually, why not do what you're doin with everything else here and just set this to SUIT_STEP by default

Reply:
Don3